api/src/main/resources/IdentifierSource.hbm.xml

```diff
@@ -23,6 +23,21 @@
 			<key column="source" not-null="true" />
 			<element column="identifier" type="string"/>
 		</set>
+                
+                <set name="autoGenerationOptions" table="idgen_auto_generation_option" lazy="true" cascade="all-delete-orphan">
+			<key column="source" update="false" />
```

Comment:
We do not need to add any of these properties for simply adding rest end points to identifier sources.

Reply:
The new properties are needed because of the purge operation. When I try to purge an IdentiferSource without the new properties I get a Referential Integrity Error. In the <set /> I am now able to declare the cascade property.

Reply:
Remove them and i see the error.

Reply:
removed

Reply:
Did you just forget to commit your changes? Because i still see what you claim to have removed.

Reply:
removed the sets.